cea/technologies/heating_coils.py

```diff
@@ -83,7 +84,15 @@ def fh(x):
             return Eq
 
         k2 = -Qcsf / mCw0
-        result = sopt.newton(fh, trc0, maxiter=100, tol=0.01) - 273
+        try:
+            result = sopt.newton(fh, trc0, maxiter=100, tol=0.01) - 273
+        except RuntimeError:
+            print('Newton optimization failed, using slower bisect algorithm...')
+            result = sopt.bisect(fh, 0, 350, xtol=0.01, maxiter=500) - 273
+
+
+        if Ta_sup_cs == Ta_re_cs:
+            print 'Ta_sup_cs == Ta_re_cs:', Ta_sup_cs
```

Comment:
this can be removed: i only added it to count how often that case happens (it happens a lot!)